sections/founder.html

```diff
@@ -0,0 +1,67 @@
+<section
+  class="bg-gradient-to-r from-gr-blue to-gr-green diagonal-pri w-screen flex justify-center -mt-36"
+>
+  <article class="md:-mt-28 max-w-[1500px]">
+    <section class="text-white px-[7%] pt-40 z-10">
+      <!-- founder div paragraphs -->
+      <div>
+        <h3 class="text-[30px] mb-5 raleway font-700">
+          Die <span class="text-primary-green">Founder</span> und ihre Vision
+          von Ojemba
+        </h3>
+        <p class="max-w-xl mb-5 text-[14px] montserrat font-500">
+          Wir werden auf dem afrikanischen Kontinent Wohlstand schaffen, indem
+          wir respektvolle gegenseitige Geschäftsbeziehungen zwischen dem Norden
+          und dem Süden des Planeten aufbauen.
+        </p>
+        <p class="max-w-xl mb-10 text-[14px] montserrat font-500">
+          Dies wollen wir erreichen, indem wir wir IT-Talente aus dem Süden mit
+          IT-Möglichkeiten aus dem Norden zusammenbringen, indem wir junge
+          afrikanische Software-Ingenieure ausbilden und ihnen anspruchsvolle
+          Jobs für Unternehmen aus dem Norden und Süden anbieten.
+        </p>
+      </div>
+      <img
+        src="../Img/Objekt 9 1.png"
+        class="mx-auto w-[60%] md:w-full z-10 relative"
+        alt="founders image"
+      />
+    </section>
+    <section class="text-white mb-64 md:mb-0 md:pb-12 bg-gr-blue diagonal-thir">
+      <div
+        class="flex md:flex-col space-x-16 md:space-x-0 pt-96 md:pt-20 pb-40 bg-[#0D2E4A] justify-center px-10 md:h-fit md:static -z-40 -mt-[30rem] py-[35rem] md:-mt-12 diagonal-sec"
+      >
+        <!-- left div -->
+        <div
+          class="max-w-[200px] md:max-w-[90%] flex flex-col space-y-5 md:space-y-0 mt-32 md:mt-0"
+        >
```

Comment:
It would be better if you don't mix rem and pixels, you can choose one and stick to it.